mod-configuration-server/src/main/resources/data/locales/currency_code_dk.json

```diff
@@ -0,0 +1,10 @@
+{
+    "id": "b873eb5a-7a50-488a-9624-d4fbc4daad51",
```

Comment:
I see that indentations differ in all samples. Please, reformat files in folder /data/locales.

Reply:
done